cowamm/balancer_cow_amms_3959044.sql

```diff
@@ -89,13 +89,42 @@ cowamms_arbitrum as (
     group by 1, 2
 ),
 
+-- on base
+cowamm_creations_base as (
+    select varbinary_substring(topic1, 1 + 12, 20) as address
+    from base.logs
+    where (
+        contract_address in (0x03362f847B4fAbC12e1Ce98b6b59F94401E4588e)
+        and topic0 = 0x0d03834d0d86c7f57e877af40e26f176dc31bd637535d4ba153d1ac9de88a7ea
+    )
+    and block_time >= cast('2024-09-01 00:00:00' as timestamp)
```

Comment:
nanonit, this can probably be at least the launch date of Base or the [creation date](https://etherscan.io/tx/0xb7c5b480520d8d3b062e516debfcd3a8bd0e940600223c19877c56df66866c3f) of this contract.